extensions/smallrye-graphql/runtime/src/main/java/io/quarkus/smallrye/graphql/runtime/SmallRyeGraphQLAbstractHandler.java

```diff
@@ -39,15 +40,15 @@ public SmallRyeGraphQLAbstractHandler(
 
     @Override
     public void handle(final RoutingContext ctx) {
-        ManagedContext requestContext = Arc.container().requestContext();
-        if (requestContext.isActive()) {
+        currentManagedContext = Arc.container().requestContext();
```

Comment:
I don't like how you write into this non-volatile field again for each request, can't we set it once and make it final, the same as `currentIdentityAssociation` and `currentVertxRequest`?